modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -282,6 +291,7 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
           node = mergeStyles(node);
         }
 
+        // TODO: Make sure all of the below handle noneditable elements correctly
```

Comment:
I still need to sanity check noneditable nodes are handled correctly by these merge functions

Reply:
To expand further on why this needs to be checked. I have made the assumption that we do not want to remove any styling from the `contenteditable="false"` element itself and/or its descendants as by definition of it being `contenteditable="false"`, we shouldn't be changing its structure